### PR TITLE
when certs are symlinks bolt complained, looking for puppet x509 stuff

### PR DESCRIPTION
**Pull request by prolixalias**
various typos in README
unused param (port)
follow links to certificate(s)

### CODE REVIEW

**Comment by MartyEwings** [comment]
@m0dular can you do a quick review of this? @prolixalias would you beable to rebase over the latest commits Thanks!

**Comment by prolixalias** [comment]
> @m0dular can you do a quick review of this? @prolixalias would you beable to rebase over the latest commits Thanks!

Sure thing Marty. Done.

File: .fixtures.yml [hunk 1]
```diff
@@ -7,11 +7,13 @@ fixtures:
...
     facts: 'https://github.com/puppetlabs/puppetlabs-facts'
-    puppet_agent: 'https://github.com/puppetlabs/puppetlabs-puppet_agent'
     provision: 'https://github.com/puppetlabs/provision'
-    bootstrap: 'https://github.com/puppetlabs/puppetlabs-bootstrap'
+    puppet_agent: 'https://github.com/puppetlabs/puppetlabs-puppet_agent'
     puppet_conf: 'https://github.com/puppetlabs/puppetlabs-puppet_conf'
     ruby_task_helper: 'https://git@github.com/puppetlabs/puppetlabs-ruby_task_helper'
     stdlib: 'https://github.com/puppetlabs/puppetlabs-stdlib'
-    deploy_pe: 'https://github.com/jarretlavallee/puppet-deploy_pe'
+  symlinks:
```
Comment (m0dular) [inline .fixtures.yml:18]: Thanks for the contribution, looks good!  I only had a question about this `symlinks` entry, what does this do?  I was able to run the unit tests with `pdk test unit` with and without this, so I think it should set up everything related to this module.  Let me know if I'm missing something, I haven't used `symlinks` in fixtures before so it's entirely possible.

Comment (prolixalias) [reply .fixtures.yml:18]: Hello! I was working around needing `influxdb_setup` type available to: `is_expected.to contain_influxdb_setup(Facter.value(:networking)['fqdn'])` in `spec/classes/init_spec.rb`

Starting to wonder if my pdk is/was misconfigured. I couldn't get it to pass without symlink to itself.

**Comment by prolixalias** [comment]
> I think you helped expose an issue where the manifest isn't passing the `port` to the `influxdb_setup` resource, thanks for finding that and removing it since it's unused. I've created an issue where we should allow for customizing this.

I'll take that one... Let's get #27 through, will start on port customization tonight off-hours

File: .fixtures.yml (hunk 1 above)
Comment (bastelfreak) [reply .fixtures.yml:18]: usually puppetlabs_spec_helper creates the symlink, thats why we removed the symlink option from a lot of modules 🤔

Comment (prolixalias) [reply .fixtures.yml:18]: Shall I resubmit without the additional symlink, not to be confused with title of PR.  :)

**Pull request closed without merging**